db/migrate/20181031182010_change_column_in_users.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnInUsers < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :users, :about_me, :about
```

Comment:
no need to create a migration to rename a column u're creating within the same PR